mvlearn/embed/kcca.py

```diff
@@ -48,11 +49,15 @@ class KCCA(BaseEmbed):
     reg : float, default = 0.1
           Regularization parameter
     decomp : string, default = 'full'
-             Decomposition type
-        - value can be only be 'full'
+             Decomposition type. ICD can reduce computation times.
```

Comment:
Name this specifically, like  "Incomplete Cholesky Decomposition (ICD) can reduce computation times" so that people know what to google at least if they want to know more about what this decomp is

Reply:
okay, also adding more about the theory in the docstrings